gclda/model.py

```diff
@@ -128,10 +128,10 @@ def __init__(self, dataset, n_topics=100, n_regions=2, alpha=.1, beta=.01,
         #   regions_sigma = (n_topics, n_regions, n_peak_dims, n_peak_dims)
         self.regions_mu = []
         self.regions_sigma = []
-        for i_topic in xrange(self.n_topics):
+        for _ in xrange(self.n_topics):
             topic_mu = []
             topic_sigma = []
-            for j_region in xrange(self.n_regions):
+            for _ in xrange(self.n_regions):
```

Comment:
`self.region_mu = [[np.zeros(shape=(1, self.n_peak_dims))] * self.n_regions] * self.n_topics
self.region_sigma = [[np.zeros(shape=(self.n_peak_dims, self.n_peak_dims))] * self.n_regions] * self.n_topics'

Like this?